src/components/Scene/Environment.component.tsx

```diff
@@ -16,5 +17,7 @@ export const Environment: FC<EnvironmentProps> = ({ environment }) => {
     };
   }, [environment]);
 
-  return <DreiEnvironment files={config.files} />;
+  const environmentIntensity = enablePostProcessing ? 12 : 4;
+
```

Comment:
This had the biggest impact on general brightness. Since the renderer now uses a correct pipeline hdr images used for enviroment lighting are handled correctly and no longer suffer from a small dynamic range. This lets us adjust the intensity here. If we use post processing this needs to be bumped up to create uniform results.